### ISSUE TITLE
explicit dependency on threads

### ISSUE THREAD

**Issue by c-cube** [gasche/bisect#1]
it seems that bisect depends on threads but doesn't indicate it properly in META or opam.

**Comment by gasche** [comment]
Yes, there is an optional thread-supporting version at <https://github.com/gasche/bisect/blob/master/src/threads/bisectThread.ml>. We can either disable it (presumably it's unused), distribute it separately, or just add the dependency. The last option seems either so that's the plan.

Out of curiosity, how did you end up hitting the issue?

**Comment by c-cube** [comment]
I hit an issue because dune failed to compile a bisected test :-)
